corehq/messaging/scheduling/tests/test_content.py

```diff
@@ -212,6 +215,28 @@ def test_sms_language_fallback(self):
             message_dict['*']
         )
 
+    def test_email_validation_valid(self):
+        email = RecipientWithEmail("test@example.com")
```

Comment:
I believe the variable name should be recipient and not email?

Reply:
58c7d914700